server/config/statementValidation.test.js

```diff
@@ -106,6 +111,10 @@ describe('lastTrainingYear', () => {
         href: '#lastTrainingYear',
         text: 'Enter the year you last attended refresher training',
       },
+      {
```

Comment:
isn't a month present in both these last two test cases?

Reply:
Yes it is, but it is month 11, while this month is 8.  When the requiredMonthIndexNotInFuture constraint determines that the year is not in the past it insists that month must be this month or earlier. The year strings 'asasas' and  `${moment().year() + 1}` are treated as being the past hence the month error message.  Perhaps there is a way of supressing month related errors when there are errors for the year.  Alternatively the month and year could be treated as one field for validation purposes...

Reply:
as discussed, we'll raise a bug for this.. it's a relatively minor issue